manim/utils/module_ops.py

```diff
@@ -99,6 +99,7 @@ def prompt_user_for_choice(scene_classes):
         name = scene_class.__name__
         console.print(f"{count}: {name}", style="logging.level.info")
         num_to_class[count] = scene_class
+        config["write_all"] = True
```

Comment:
It seems a bit odd to me that this is set inside of the for loop, I'd prefer if it was set outside of it.

Reply:
> Oh yeah, I don't know why I put it at the end inside of a loop, my bad. I'll fix right now.